### ISSUE TITLE
Minecraft Update 1.19

### ISSUE THREAD

**Issue by ezfe** [ThatGuySam/doesitarm#951]
**The official name of the app**
Minecraft

**Is there a supported version available on a stable release channel?**
Yes

**Proposed New App Status**
Native

**Related Issue Tracker Links or discussion**
N/A

**An Official App Download Page**
https://www.minecraft.net/en-us/download

**Additional context**
Minecraft 1.19 runs native on Apple Silicon if a native Java runtime (such as [this](https://www.azul.com/downloads/?version=java-17-lts&os=macos&architecture=arm-64-bit&package=jre)) is selected from within the Minecraft app settings. No additional configuration is needed.

**Full Screenshot with the App and Activity Monitor(if self reporting) or [Silicon Info](https://github.com/billycastelli/Silicon-Info)**
<img width="1440" alt="Screen Shot 2022-06-12 at 9 34 06 AM" src="https://user-images.githubusercontent.com/1449259/173235768-f80cb26e-3127-466e-b623-265950b0b179.png">

**Comment by ThatGuySam** [comment]
Thanks for the update! 

So you're saying if you set up an Apple SIlicon Native Java Runtime it's Native, but the standard version distributed through the Minecraft site is not?

**Comment by ezfe** [comment]
Yes, the user has to download a native JRE and modify the settings within the launcher to use that JRE, but then it will run natively.

The launcher itself does not run natively.
